reflac

```diff
@@ -86,6 +95,9 @@ recompress()
                 mv -f "$flac.new" "$flac"
             fi
             rm -f -- "$TMPDIR/$(basename -s flac -- "$flac")tag"
+            if [ $HAS_COVER_ART -eq 1 ]; then
+                rm -f -- "$TMPDIR/$(basename -s flac -- "$flac")img"
+            fi
```

Comment:
`rm -f` doesn't fail when a file doesn't exist, it would be safe to remove the `if` block surrounding it. Perhaps just combine it with the previous `rm` command.